ansible/roles/csm.sbps.lio_config/files/provision_iscsi_server.sh

```diff
@@ -41,12 +41,17 @@ function add_server_target()
 {
         TARGET_SERVER_IQN="${IQN_PREFIX}$1"
         NMN_IP="$2"
-        HSN_IP="$3"
-        CMN_IP="$4"
+        CMN_IP="$3"
         targetcli "/iscsi create $TARGET_SERVER_IQN" &> /dev/null
         targetcli "/iscsi/${TARGET_SERVER_IQN}/tpg1/portals delete ip_address=0.0.0.0 ip_port=3260" &> /dev/null
         targetcli "/iscsi/${TARGET_SERVER_IQN}/tpg1/portals create ${NMN_IP}" &> /dev/null
-        targetcli "/iscsi/${TARGET_SERVER_IQN}/tpg1/portals create ${HSN_IP}" &> /dev/null
+
+        if [[ ! -z $HSN_IP ]]
+        then
+          HSN_IP="$4"
```

Comment:
This seems reversed, it's checking `$HSN_IP` before it has been set.  Shouldn't it be:
```
HSN_IP="$4"
if [[ -n $HSN_IP ]]
then
```